xapian-core/tests/api_queryparser.cc

```diff
@@ -1726,6 +1726,39 @@ DEFINE_TESTCASE(qp_range4, !backend) {
     return true;
 }
 
+static const test test_unitrange1_queries[] = {
+    { "financial report size:100K..1M", "((financial@1 OR report@2) FILTER VALUE_RANGE 1 \\xe0&@ \\xe04)" },
+    { NULL, NULL }
+};
+
+// Test chaining of UnitRangeProcessor class.
```

Comment:
This isn't testing chaining of this class, it's just a simple feature test for the class.

Reply:
Updated comment.